tests.yaml

```diff
@@ -0,0 +1,239 @@
+# same one you would call it at a conference. If the program is only known
+# inside your company, prepend its name with the company name.
+#
+# The "home" property should point to the canonical, upstream location, and is
+# intended for identification and as a reference to more information on the
+# test, rather than a pointer to exact code being executed. The latter should
+# be described within test objects submitted to kcidb instead.
+#
+# E.g. the Linux Test Project test suite *should* be here under the
+# widely-known name "ltp". However, while its own tests, such as "chdir01" or
+# "clone03" are welcome to be reported to kcidb as "ltp.chdir01" or
+# "ltp.clone03", only the "ltp" test should be in this catalog. The LTP
+# project's home page or its main upstream repository should be referred to in
+# the "home" property, and not e.g. the specific code being executed, because
+# code versions will differ between test result origins and even executions.
+#
+# To decide where to place your test, follow organizational patterns already
+# in the file, and discuss any non-obvious cases when submitting the change.
+#
+# The objective of organization is to simplify locating tests as much as
+# possible, which can be helped by observing the following rules of thumb.
+#
+#   * Have only one organizational dimension per "tests" dictionary.
+#     E.g. if a "tests" dictionary has subsystem names as keys, then it
+#     shouldn't have device names there as well.
+#   * Avoid creating too many small categories. Instead, create a "misc"
+#     category and put all the miscellaneous tests there, together. Once the
+#     "misc" category accumulates enough certain tests for a category of their
+#     own, move them out.
+#   * Avoid placing programs together with categories in the same "tests"
+#     dictionary, if they could go into "misc" category instead.
+#
+# You can use the "kcidb-tests-validate" tool to check if this file is valid.
+# Like this:
+#
+#     kcidb-tests-validate < tests.yaml
+#
+# If you add the -u/--urls option, it will also check if all URLs are
+# reachable:
+#
+#     kcidb-tests-validate --urls < tests.yaml
+#
+# You can also use the "kcidb-tests-list" tool to print a flat list of known
+# test programs:
+#
+#   kcidb-tests-list < tests.yaml
+#
+title: The top-level "test" signifying testing in general
+description: |
+    Report results on this test when you don't have any more details on your
+    testing, and/or when you want to signal your testing for the build has
+    concluded.
+tests:
+    storage:
+        title: Storage subsystem tests
+        tests:
+            fio:
+                title: Flexible I/O tester
+                home: https://github.com/axboe/fio
+            blktests:
+                title: Test framework for the Linux kernel block layer and storage stack
+                home: https://github.com/osandov/blktests
+            redhat_scsi_vpd:
```

Comment:
It would seem better imho to call these CKI tests `cki_*` rather than `redhat_*`.

Reply:
I see what you mean, but those tests predate CKI by a wide margin. I don't think there are any tests which are CKI-specific (as in created to run in CKI) here, or maybe one or two only. They only have "redhat_" in front of them, because they're only known inside Red Hat. Furthermore, you shouldn't have too much of a hard time running them outside CKI, and it's our target to improve that.

So, I'd rather keep "redhat_" in the names, as that would be more to the point.

Reply:
OK fine, I thought of this because the projects are hosted under `CKI-project` on GitHub.  Maybe if they had been in another more general Red Hat organisation it would have seemed more natural to call them `redhat_*`.  But I guess you guys are the authority on this topic, and as long as the tests aren't completely tied to CKI and Beaker then great :)

Reply:
Well, somebody **had** to get them outside of the company, and it just so happened that we needed that the most :grin:

They're tied most to Beaker, and much less so to CKI, if at all. Beaker is much, much older. Still we're working on getting them untied as much as possible.